db/migrate/20201029183949_add_setup_intent_fields_to_customer.rb

```diff
@@ -0,0 +1,6 @@
+class AddSetupIntentFieldsToCustomer < ActiveRecord::Migration
+  def change
+    add_column :customers, :gateway_setup_intent_id, :string
+    add_column :customers, :gateway_setup_intent_client_secret, :string
```

Comment:
ah, nice!
in the credit card we have gateway_customer_profile_id and gateway_payment_profile_id, these are not stripe specific fields. Can we rename these setup intents to something that is not stripe specific? Something related to recurring payments? gateway_recurring_payment_id and gateway_recurring_payment_client_secret?

Reply:
5d43910e10c77b4816f62a4025ab49e383ca83be 👍